index.js

```diff
@@ -353,17 +359,17 @@ class Plugin extends MiniAccountsPlugin {
 
       await this._watcher.watch(channel)
       await this._registerAutoClaim(account)
-      debug('registered payment channel. account=', account.getAccount())
+      this._log.trace('registered payment channel. account=', account.getAccount())
     }
 
     if (fundChannel) {
       if (new BigNumber(util.xrpToDrops(account.getPaychan().amount)).lt(MIN_INCOMING_CHANNEL)) {
-        debug('denied outgoing paychan request; not enough has been escrowed')
+        this._log.debug('denied outgoing paychan request; not enough has been escrowed')
         throw new Error('not enough has been escrowed in channel; must put ' +
           MIN_INCOMING_CHANNEL + ' drops on hold')
       }
 
-      debug('an outgoing paychan has been authorized for ', account.getAccount(), '; establishing')
+      this._log.trace('an outgoing paychan has been authorized for ', account.getAccount(), '; establishing')
```

Comment:
this might be good to see as info because it spends a significant amount of XRP and doesn't occur super frequently